src/helpers/sendMessage.js

```diff
@@ -0,0 +1,34 @@
+import sgMail from '@sendgrid/mail';
+import dotenv from 'dotenv';
+
+dotenv.config();
+sgMail.setApiKey(process.env.API_KEY);
+const { SENDGRID_EMAIL } = process.env;
+const sendMessage = async (userEmail, token, clientURL) => {
+  try {
```

Comment:
use meaningful variable because we will have a lot of sending messages variables